cico_deploy.sh

```diff
@@ -1,4 +1,37 @@
 #!/bin/bash
+set -eu
 
-# Placeholder for now, will be used to docker push
-# a freshly built Che container set
+# Source build variables
+cat jenkins-env | grep -e ^CHE_ > inherit-env
+. inherit-env
+. config 
+
+# Install oc client
+yum install -y centos-release-openshift-origin
+yum install -y origin-clients
+
+# Login
+oc login -u "${CHE_OPENSHIFT_USERNAME}" -p "${CHE_OPENSHIFT_PASSWORD}"
+
+# Create or update template
+oc create -f che.json >/dev/null 2>&1 || oc replace -f che.json >/dev/null 2>&1
+
+# Check if deploymentConfig is already present
+oc get dc che-host > /dev/null 2>&1
+if [[ $? -eq 0 ]]; then
+    # Cleanup the project
+    oc project ${CHE_OPENSHIFT_PROJECT}
```

Comment:
maybe this is too late, since you are changing the project name after having checked for the che deployment - i presume you want to only do actions on the project you had checked in ?